src/types/props/literal.ts

```diff
@@ -3,11 +3,11 @@ import { Node } from '../nodes/baseNodes';
 const typeString = 'LiteralNode';
 
 export interface LiteralNode extends Node {
-	value: any;
+	value: unknown;
```

Comment:
Why was this change needed?

Reply:
To add type safety. This could've been used earlier to catch this bug.

Reply:
Ah, right, of course.